components/brave_rewards/resources/tip/components/form_submit_button.tsx

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+import * as style from './form_submit_button.style'
+
+// ===============
+//  CSS variables
+// ===============
+//
+// --icon-margin-top
+
+interface Props {
+  onClick: () => void
+  children: React.ReactNode
+}
+
+let hasFocused = false
+
+function maybeFocus (button: HTMLButtonElement | null) {
+  if (button && !hasFocused) {
+    hasFocused = true
+    button.focus()
+  }
+}
+
+export function FormSubmitButton (props: Props) {
+  return (
+    <style.root>
+      <button
```

Comment:
This is the only button that seems to not have a focus state, so difficult to use without mouse.

Reply:
Focus state has been fixed. Thanks!